src/language/mod.rs

```diff
@@ -167,6 +167,12 @@ impl Language {
 	}
 
 	/// Get the index of the word in the word list.
+	///
+	/// WARNING:
+	///
+	/// Indices returned by this function start with `0`, while indices start with `1` in a lookup against [BIP39 on github](https://github.com/bitcoin/bips/blob/master/bip-0039/english.txt) (and most wallet docs).
```

Comment:
What do you mean by "a lookup against bip39 on github"? Do you mean the numbers show by the github UI? If so those are line numbers, line numbers always start from one. Indexes start from 0.